### ISSUE TITLE
Question about the balanced softmax loss

### ISSUE THREAD

**Issue by cledor8712** [jiawei-ren/BalancedMetaSoftmax-Classification#13]
Hi, thanks for your research and sharing codes. It helped me a lot.

I have a question about the balanced softmax.

I understand that balanced softmax uses prior distributions during the training phase.

For example, cross entropy + prior distribution log (I'm not sure it is the right explanation)

Then, should we use same balanced softmax loss during the test phase, too?

Or just use the normal cross entropy?

Intuitively, test distribution is different from the train distribution, so I thought we don't need to use the balanced softmax loss during 

the test phase.

Thank you!

**Comment by jiawei-ren** [comment]
> Intuitively, test distribution is different from the train distribution, so I thought we don't need to use the balanced softmax loss during the test phase.

Yes, we don't need balanced softmax in the test phase. Sorry for the late reply.
